bounties/pip/caliban/1/vulnerability.json

```diff
@@ -0,0 +1,50 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-12-24",
+    "AffectedVersionRange": "*",
+    "Summary": "Arbitrary Code Execution",
+    "Contributor": {
+      "Discloser": "B3EF",
+      "Fixer": ""
+    },
+    "Package": {
+      "Registry": "pip",
+      "Name": "caliban",
+      "URL": "https://github.com/google/caliban.git"
+    },
+    "CWEs": [
+      {
+        "ID": "94",
+        "Description": "Arbitrary Code Execution"
+      }
+    ],
+    "CVSS": {
+      "Version": "3.1",
+      "AV": "",
+      "AC": "",
+      "PR": "",
+      "UI": "",
+      "S": "",
+      "C": "",
+      "I": "",
+      "A": "",
+      "E": "",
+      "RL": "",
+      "RC": "",
+      "Score": "8.5"
+    },
+    "CVEs": [""], 
+    "Repository": {
+      "URL": "https://github.com/google/caliban.git",
+      "Codebase": ["PYTHON"],
+      "Owner": "google",
+      "Name": "caliban"
+    },
+    "Permalinks": ["https://github.com/google/caliban/blob/56f96e7e05b1d33ebdebc01620dc867f7ec54df3/caliban/platform/gke/util.py#L790"], 
```

Comment:
Change with https://github.com/google/caliban/blob/master/caliban/platform/gke/util.py#L790 :smile:

Reply:
Changed as you sugested @Mik317